packages/get-starknet/src/rpcs/add-invoke.test.ts

```diff
@@ -0,0 +1,52 @@
+import { mockWalletInit, createWallet } from '../__tests__/helper';
+import { MetaMaskSnap } from '../snap';
+import { formatCalls } from '../utils/helper';
+import { WalletAddInvokeTransaction } from './add-invoke';
+
+describe('WalletAddInvokeTransaction', () => {
+  const mockInvokeTransaction = (expectedResult) => {
+    const spy = jest.spyOn(MetaMaskSnap.prototype, 'execute');
+    spy.mockResolvedValue(expectedResult);
+    return spy;
+  };
+
+  const prepareInvokeTransaction = (expectedResult) => {
+    const wallet = createWallet();
+    const { initSpy: walletInitSpy } = mockWalletInit({});
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    const invokeTransactionSpy = mockInvokeTransaction(expectedResult.transaction_hash);
```

Comment:
```suggestion
    const invokeTransactionSpy = mockInvokeTransaction(expectedResult);
```

the result is {  transaction_hash: "xxx" }, but not a string